src/main/java/roomescape/service/ReservationService.java

```diff
@@ -0,0 +1,43 @@
+package roomescape.service;
+
+import org.springframework.stereotype.Service;
+import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+import roomescape.dto.ReservationCreateRequestDto;
+import roomescape.dto.ReservationResponseDto;
+import roomescape.repository.ReservationRepository;
+import roomescape.repository.ReservationTimeRepository;
+
+import java.util.List;
+
+@Service
+public class ReservationService {
+
+    private final ReservationRepository reservationRepository;
+    private final ReservationTimeRepository reservationTimeRepository;
+
+    public ReservationService(ReservationRepository reservationRepository, ReservationTimeRepository reservationTimeRepository) {
+        this.reservationRepository = reservationRepository;
+        this.reservationTimeRepository = reservationTimeRepository;
+    }
+
+    public ReservationResponseDto createReservation(ReservationCreateRequestDto dto) {
+        ReservationTime reservationTime = reservationTimeRepository.findById(dto.timeId());
+        Reservation requestReservation = dto.toEntityWith(reservationTime);
+        Reservation newReservation = reservationRepository.save(requestReservation);
+        ReservationResponseDto reservationResponseDto = ReservationResponseDto.from(newReservation, newReservation.time());
+        return reservationResponseDto;
```

Comment:
변수 할당없이 아래와 같이 바로 반환해도 되겠네요 :)

```suggestion
        return ReservationResponseDto.from(newReservation, newReservation.time());
```

Reply:
좋습니다! 수정하였습니다.